include/uapi/sound/sof/tokens.h

```diff
@@ -133,6 +132,7 @@
 
 /* Processing Components */
 #define SOF_TKN_PROCESS_TYPE                    900
+#define SOF_TKN_PROCESS_INIT_BLOB_HAS_OUT_FMT   901
```

Comment:
Any hint on the why?

Reply:
I add comment in the variable in snd_sof_process struct, I think we don't need comments at both places, right?

Reply:
I meant for this change...
Originally you used completely different token.

Reply:
and you obviously need to provide a new commit message for the original commit as the `SOF_TKN_COMP_PAYLOAD_WITH_OUTPUT_FMT` is gone with this "fixup"

Reply:
if this flag is true, it means there is output format in the init blob, we need to append the format to the init blob (after base module config).

module init blob construct steps:
mandatory base module config + [optional output format] + [optional module specific extension from topology]

Reply:
> I meant for this change... Originally you used completely different token.

@ujfalusi I think this is OK. luckily this token isnt currently used in any of our topologies. The change is needed becaue we're moving away from being a COMP_TOKEN to a PROCESS_TKN

Reply:
> and you obviously need to provide a new commit message for the original commit as the `SOF_TKN_COMP_PAYLOAD_WITH_OUTPUT_FMT` is gone with this "fixup"

I will add some explanation to why I do a rename, and why we move it from a comp token to process module token.